tests/composite_viscous_outputs.cc

```diff
@@ -39,7 +39,7 @@ void f(const aspect::SimulatorAccess<dim> &simulator_access,
   const unsigned int composition = 0;
   const std::vector<double> volume_fractions = {1.};
   const std::vector<double> phase_function_values = std::vector<double>();
-  const std::vector<unsigned int> n_phases_per_composition = std::vector<unsigned int>(1);
+  const std::vector<unsigned int> n_phase_transitions_per_composition = std::vector<unsigned int>(0);
```

Comment:
Why 0 instead of 1?

Reply:
Ah, thank you. I initially wrote (1, 0) and deleted the wrong number. In this case it shouldn't make a difference, but I've changed it anyway.